src/main/java/com/flansmod/common/network/PacketIncreaseZoom.java

```diff
@@ -0,0 +1,42 @@
+package com.flansmod.common.network;
```

Comment:
This should be merged with PacketDecreaseZoom with a boolean, having two packets is unneccessary